solidity/deploy/07_deploy_bridge_governance.ts

```diff
@@ -0,0 +1,41 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre
+  const { deploy } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  const Bridge = await deployments.get("Bridge")
+
+  const BridgeGovernanceParameters = await deployments.deploy(
+    "BridgeGovernanceParameters",
+    {
+      from: deployer,
+      log: true,
+    }
+  )
+
+  const GOVERNANCE_DELAY = 604800 // 1 week
+
+  const BridgeGovernance = await deploy("BridgeGovernance", {
+    contract: "BridgeGovernance",
```

Comment:
As the deployment and contract names are the same we can skip the `contract` property.

```suggestion
  const BridgeGovernance = await deploy("BridgeGovernance", {
```

Reply:
part of [40d1f5f](https://github.com/keep-network/tbtc-v2/pull/329/commits/40d1f5f0583534c0683a5f0b871608c216b85900)